frontend/src/ches/Performance.jsx

```diff
@@ -0,0 +1,37 @@
+import React,{Component} from 'react';
+
+
+class Performance extends Component {
+
+  render() {
+
+    return (
+      <div className="container" id="maincontainer">
+        <div id="mainrow" className="row">
```

Comment:
Is this "mainrow" ID needed? Referenced anywhere?

Reply:
I don't think it is needed (in this case), but it's used on all of the other screens, basically used to keep a min-height for the contents. It also has styles for the tabs contained on the screen.

Reply:
Ah, yeah I see the style in  index.scss. Probably best to change to be a **class** then, but not too important.

Reply:
thanks. i can try and do this